appveyor.yml

```diff
@@ -0,0 +1,124 @@
+platform: x64
+environment:
+ matrix:
+  - DC: dmd
+    DVersion: nightly
```

Comment:
Would be great if we could stick to the established `dmd-nightly` et.al. names.

Reply:
Seems like powershell has the necessary wildcard switch.
https://technet.microsoft.com/en-us/library/ff730937.aspx#EGC